packages/mui-joy/src/Breadcrumbs/Breadcrumbs.test.tsx

```diff
@@ -30,11 +30,14 @@ describe('<Breadcrumbs />', () => {
 
       expect(getByRole('navigation')).to.have.class(classes.sizeMd);
     });
-    ['sm', 'md', 'lg'].forEach((size) => {
+    const sizes = ['sm', 'md', 'lg'] as const;
+    sizes.forEach((size) => {
```

Comment:
```suggestion
    (['sm', 'md', 'lg'] as const).forEach((size) => {
```

For consistency with other tests